### PR TITLE
Feature/add heartbeat

### PR DESCRIPTION
**Pull request by santiagonoguez**
I changed this code on common-display-module so local-messaging-module can leverage most of heartbeat timer code, but customize the function that performs the actual broadcasting.

### CODE REVIEW

File: heartbeat.js [hunk 1]
```diff
@@ -5,6 +5,20 @@ const DEFAULT_HEARTBEAT_INTERVAL = 4;
 
 let timerId = null;
 
+const DEFAULT_BROADCAST_ACTION = message =>
+{
```
Comment (andrecardoso) [inline heartbeat.js]: Wouldn't it be better to start this block in the previous line, right after the arrow?

File: test/unit/heartbeat.js [hunk 2]
```diff
@@ -29,4 +32,20 @@ describe("Heartbeat - Unit", ()=>
     })
   });
 
+  it("should let change the broadcast action", done => {
+    heartbeat.setBroadcastAction(message =>
+    {
```
Comment (andrecardoso) [inline test/unit/heartbeat.js]: Same as above? Wouldn't it be better to start this block right after the arrow? I see you are doing that below on the `startHearbeatInterval` call. Why not do the same here?

File: heartbeat.js [hunk 3]
```diff
@@ -5,6 +5,20 @@ const DEFAULT_HEARTBEAT_INTERVAL = 4;
...
+  // nested require to avoid circular dependency problem.
+  const common = require("./common");
+
+  common.broadcastMessage(message);
+}
+
+let broadcastAction = DEFAULT_BROADCAST_ACTION;
+
+function setBroadcastAction(action) {
+  broadcastAction = action ? action : DEFAULT_BROADCAST_ACTION;
```
Comment (andrecardoso) [inline heartbeat.js]: Maybe add a check here to ensure the parameter is a function?

File: test/unit/heartbeat.js (hunk 2 above)
Comment (santiagonoguez) [reply test/unit/heartbeat.js]: @andrecardoso thanks !!! styles corrected, I'm still reforming from previous style conventions !

**Pull request merged**